plugins/modules/ldap_search.py

```diff
@@ -91,6 +91,7 @@
   register: ldap_group_gids
 """
 
+# @FIXME RV 'results' is meant to be used when 'loop:' was used with the module.
```

Comment:
I'm not sure about this comment. I think `results` is a perfectly fine return value for modules. If you disagree, maybe this should be a discussion on the forum first?

Reply:
The more logical solution for me would be to change the common RV from `results` to, say, `loop_results` or similar. I suspect the core team will not be keen to go for it, but anyways, I will raise the issue in the forum and take it from there.

Are you OK with this PR proceeding as is?

Reply:
Yes, let's merge this. We can update the comments later on.